src/modules/session/thunks.ts

```diff
@@ -72,21 +72,10 @@ export const handlelLoginError = createAsyncThunk(
   "session/handlelLoginError",
   // eslint-disable-next-line
   async (error: any, thunkApi) => {
-    let errorMessage;
-
-    switch (error?.status) {
-      case 401:
-        errorMessage = "The password is invalid";
-        break;
-      case 404:
-        errorMessage = "No account was found with the email provided";
-        break;
-      case 409:
-        errorMessage = "The email for this account is already in use";
-        break;
-      default:
-        errorMessage = "An error occurred while logging in";
-    }
+    const errorMessage =
+      error?.status === 409
+        ? "The email for this account is already in use"
+        : "Email or password is incorrect";
```

Comment:
Removing the more detailed error message and making a generic error message instead